tests/integration/test_storage_kafka/test_schema_registry_skip_bytes.py

```diff
@@ -0,0 +1,159 @@
+"""Tests for kafka_schema_registry_skip_bytes setting"""
+
+import logging
+import pytest
+from helpers.cluster import ClickHouseCluster
+from helpers.kafka.common_direct import *
+import helpers.kafka.common as k
+from helpers.test_tools import TSV
+from contextlib import contextmanager
+
+
+cluster = ClickHouseCluster(__file__)
+instance = cluster.add_instance(
+    "instance",
+    main_configs=["configs/kafka.xml"],
+    user_configs=["configs/users.xml"],
+    with_kafka=True,
+    with_zookeeper=True,
+    stay_alive=True,
+    macros={
+        "kafka_broker": "kafka1",
+        "kafka_format_json_each_row": "JSONEachRow",
+    },
+    clickhouse_path_dir="clickhouse_path",
+)
+
+
+@pytest.fixture(scope="module")
+def kafka_cluster():
+    try:
+        cluster.start()
+        kafka_id = instance.cluster.kafka_docker_id
+        print(("kafka_id is {}".format(kafka_id)))
+        yield cluster
+    finally:
+        cluster.shutdown()
+
+
+@pytest.fixture(autouse=True)
+def kafka_setup_teardown():
+    k.clean_test_database_and_topics(instance, cluster)
+    yield  # run test
+
+
+def test_schema_registry_skip_bytes_valid_values(kafka_cluster):
+    """Test that valid values for kafka_schema_registry_skip_bytes are accepted"""
+    topic_name = f"skip_bytes_valid_{k.random_string(6)}"
+    
+    # Test valid values: 0, 19 (AWS Glue), and 255 (max)
+    valid_values = [0, 19, 255]
+    
+    for skip_bytes in valid_values:
+        table_name = f"kafka_table_{skip_bytes}_{k.random_string(4)}"
+        
+        # Create table with the skip_bytes setting
+        create_query = k.generate_new_create_table_query(
+            table_name,
+            "key String, value String",
+            topic_list=topic_name,
+            consumer_group=f"{topic_name}_group_{skip_bytes}",
+            settings={"kafka_schema_registry_skip_bytes": skip_bytes}
+        )
+        
+        # This should succeed without errors
+        instance.query(create_query)
+        
+        # Verify the table was created
+        result = instance.query(f"SHOW TABLES LIKE '{table_name}'")
+        assert table_name in result
+        
+        # Clean up
+        instance.query(f"DROP TABLE IF EXISTS {table_name}")
```

Comment:
I am not sure if this makes too much sense. Technically the only addition it gives to `test_schema_registry_skip_bytes_message_processing` is checking that the table shows up in `SHOW TABLES` query. The kafka tests are already too long, let's not add test that check only a very specific, very general thing.